extensions/sql-database-projects/src/tools/autorestHelper.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { DoNotAskAgain, Install, nodeButNotAutorestFound, nodeNotFound } from '../common/constants';
+import * as utils from '../common/utils';
+import { DBProjectConfigurationKey } from './netcoreTool';
+import { ShellExecutionHelper } from './shellExecutionHelper';
+
+const autorestPackageVersion = '0.0.2'; // latest version of AutoRest.Sql package on npm
```

Comment:
can the latest version be pulled in or is it safer to use this version and then update it when needed?

Reply:
I can set it to @latest, but maybe that should be overrideable via setting just in case...